pkg/klusterlet/clusterclaim/clusterclaimer.go

```diff
@@ -107,7 +108,10 @@ func init() {
 		clusterv1beta1.LabelCloudVendor,
 		clusterv1beta1.LabelKubeVendor,
 		clusterv1beta1.LabelManagedBy,
-		clusterv1beta1.OCPVersion)
+		clusterv1beta1.OCPVersion,
+		clusterv1beta1.OCPVersionMajor,
+		clusterv1beta1.OCPVersionMajorMinor,
```

Comment:
This is the root cause of the clusterclaim not be created by `syncLabelsToClaims`.
/assign @zhiweiyin318 
/cc @gparvin